### DISCUSSION TITLE
Do we still have the old-style `parse_snippet` syntax?

### DISCUSSION THREAD

**Discussion by nyngwang** [L3MON4D3/LuaSnip#902] [Q&A] [answered]
So now I just get familiar with the current approach:

<img src="https://github.com/L3MON4D3/LuaSnip/assets/24765272/ca5c619e-82ec-4b66-a5d1-0ef0f120e1d6" width="300">

but I still want to finish [the video](https://youtu.be/Dn800rlPIho) by TJ, and I noticed that he used a different way to define the snippet as below. My question is: do we still support this old way to define a snippet? i.e. I can put some `$`-sign prefixed numbers in the snippet string. Or is it completely replaced by the `fmt`-`{}` approach now?

<img src="https://github.com/L3MON4D3/LuaSnip/assets/24765272/8cb3f758-1119-4af0-b0dd-e55adedea333" width="400">

**Comment by L3MON4D3** [comment, marked as answer]
`parse_snippet` is still supported, it should be at the exact same place too
